BIT285_SpicyShrimps/BIT285_SpicyShrimps/Views/Login/TeacherLogin.cshtml

```diff
@@ -70,8 +70,8 @@
                                         <input asp-for="Password" class="form-control" />
                                         <span asp-validation-for="Password" class="text-danger"></span>
                                     </div>
-                                   
-                                    <div class="form-group">
+   @*added p tag for space between create button and password*@                                
```

Comment:
Change the p tag to a br, or let this be changed in the CSS. Don't use semantic html elements for structure on a page, this confuses screenreaders.